dmn-juel-evaluation-provider/src/test/java/org/powerflows/dmn/engine/evaluator/expression/provider/sample/MethodSource.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2018-present PowerFlows.org - all rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.powerflows.dmn.engine.evaluator.expression.provider.sample;
```

Comment:
instead of copying this I'd create a separate module with test resources, containing a MethodSource class and all dmn, yaml resources that could be reused in IO & DSL modules tests...

Reply:
For now the only one required to duplicate is MethodSource.java so maybe let's leave it as it is?

Reply:
Also extracted to separated module.